Backpack/BarChart/Classes/BarChartBar.swift

```diff
@@ -0,0 +1,181 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+
+@objcMembers
+@objc(BPKBarChartBar)
+public final class BPKBarChartBar: UIControl {
+
+    let backgroundView: UIView = UIView()
+    let barView: UIView = UIView()
```

Comment:
I was thinking we'd do this with layers instead of dedicated views. It would make for a simpler view hierarchy

Reply:
That seems like it would involve some horrible calculations in code?

Reply:
A single percentage calculation?